osmosis-apidb/src/main/java/org/openstreetmap/osmosis/apidb/v0_6/ApidbWriter.java

```diff
@@ -111,17 +112,19 @@
     private static final String LOAD_CURRENT_NODE_TAGS =
     	"INSERT INTO current_node_tags SELECT node_id, k, v FROM node_tags WHERE node_id >= ? AND node_id < ?";
 
+    public static final String WHERE_WAY_ID_AND_WAY_ID = " WHERE way_id >= ? AND way_id < ?";
```

Comment:
This causes a checkstyle failure due to being public and missing javadoc.  It shouldn't be public.